### PR TITLE
rsdl-js: doc comments

### PR DESCRIPTION
**Pull request by ralfhandl**
PoC for #247, see https://github.com/oasis-open/odata-rapid/blob/8d5e7362e14161c1373b83a496ebc108dd02b470/tools/rsdl/rsdl-js/test/parser.test.js#L402-L474

### CODE REVIEW

File: tools/rsdl/rsdl-js/test/parser.test.js [hunk 1]
```diff
@@ -396,6 +398,80 @@ describe("Parse correct RSDL", () => {
       }
     );
   });
+
+  it("Documentation comments", () => {
+    assert.deepStrictEqual(
+      parse(`## good type
+             ##
+             ## has nice properties
```
Comment (ralfhandl) [inline tools/rsdl/rsdl-js/test/parser.test.js:406]: Also test empty lines between doc comments

Comment (ralfhandl) [reply tools/rsdl/rsdl-js/test/parser.test.js:406]: And plain comments interspersed within a block of doc comments

File: tools/rsdl/rsdl-js/lib/parser.js [hunk 2]
```diff
@@ -69,20 +69,29 @@ class MyListener extends rsdlListener {
...
     }
 
-    const term = this.normalizeTermName(ctx.qualifiedName().getText());
-    this.annotatable[0].target[
-      `${this.annotatable[0].prefix}@${term}`
-      //TODO: parse annotation value
-    ] = this.annotation[0].value;
+    const docComment = ctx.DOC_COMMENT();
+    if (docComment) {
+      const name = `${this.annotatable[0].prefix}@Org.OData.Core.V1.Documentation`;
```
Comment (mikepizzo) [inline tools/rsdl/rsdl-js/lib/parser.js:74]: Should be Org.OData.Core.V1.Description

**Review by mikepizzo — CHANGES_REQUESTED** [review]
I like it, but annotation should be Org.OData.Core.V1.Description rather than Org.OData.Core.V1.Documentation

File: tools/rsdl/rsdl-js/test/parser.test.js (hunk 1 above)
Comment (ralfhandl) [reply tools/rsdl/rsdl-js/test/parser.test.js:406]: Done

**Comment by ralfhandl** [comment]
> I like it, but annotation should be Org.OData.Core.V1.Description rather than Org.OData.Core.V1.Documentation

Done

**Review by mikepizzo — APPROVED** [review]

**Review by chrisspre — APPROVED** [review]
